### ISSUE TITLE
ADA Color Contrast Issues (Major)

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/library_website#321]
Issues are grouped under page section. Each section has the ADA Severity plus the Development cost (eg: Major / Easy).

## Main Navigation (Major / Easy)
Issues: 115173, 115296, 115384
### Problem
- The contrast ratio between link or button text and its background is not at least 4.5:1 on hover and/or on focus.
- All the Links in the submenu expansion lacks 4.5:1 contrast ratio on hover.
Current Contrast Ratio : 3.4:1
- [Screenshot example](https://wsassure-reports.dequecloud.com/api/v1/file/0471cdee-1cca-11ea-8007-8b35ce2791a5)

### Solution
- Fix this issue by adjusting the color of the hovered or focused link or button text and/or background to increase the contrast to at least 4.5 to 1.

## Default text link (Major / Easy)
Issues: 115284, 115589, 135778, 136732, 137564, 137677
### Problem
- The contrast ratio between default link text and surrounding text is not at least 3:1. (An additional differentiator appears on hover.)
- Color alone MUST NOT be used to distinguish links from surrounding text unless the color contrast between the link and the surrounding text is at least 3:1 and an additional differentiation (e.g. underline, outline, etc.) is provided when the link is hovered or receives focus.
- [Screen shot example](https://wsassure-reports.dequecloud.com/api/v1/file/0df949a6-2d31-11ea-967a-dbc5b725873c)
- Also seen on [Hours page](https://wsassure-reports.dequecloud.com/api/v1/file/12417e5e-2b9f-11ea-ab54-f364210b8279), [Featured Expert widget](https://wsassure-reports.dequecloud.com/api/v1/file/14df83aa-2ba8-11ea-a158-8f5c99e4aca0) (homepage), and various other places. Check Deque's issue links to verify all places found are resolved.

### Solution (Either or both)
- Add an additional indicator (e.g. underline, outline, etc) to the link text when it is in its default state.
- increase the contrast between the link text and the body text so that the ratio is at least 3.0 to 1. (Be sure to keep the additional indicator (e.g. underline, outline, etc) on hover.)

## Default text link hover (Major / Easy)
Issues: 135775, 136196, 136624, 137143, 137169, 137562, 137679, 137939, 137940, 138010, 138048
### Problem
- The contrast ratio between link or button text and its background is not at least 4.5:1 on hover and/or on focus.
- [Screenshot example](https://wsassure-reports.dequecloud.com/api/v1/file/b8733098-306c-11ea-a672-afafbb08d79e)

### Solution
- Fix this issue by adjusting the color of the hovered or focused link or button text and/or background to increase the contrast to at least 4.5 to 1.
